### PR TITLE
Replace class with className #178

### PR DESCRIPTION
**Pull request by kenkirito**
## Issue that this pull request solves

 Closes: # (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

HITK Tech Community and 49 more pages - Personal - Microsoft​ Edge 05-01-2021 02_26_05 PM

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/src/components/terms/terms.js [hunk 1]
```diff
@@ -13,38 +13,36 @@ const Terms = () => {
         </div>
       </div>
       <div className="agreement">
-          <div class="row">
```
Comment (mohit355) [inline frontend/src/components/terms/terms.js:16]: Only class keyword changed required to className. No need for removing the classes.

File: frontend/src/components/terms/terms.css [hunk 2]
```diff
@@ -9,7 +9,13 @@
     height: 50vh;
     opacity: 0.8;
 }
-
+.terms_space {
```
Comment (mohit355) [inline frontend/src/components/terms/terms.css:12]: No need for this class

File: frontend/src/components/terms/terms.css [hunk 3]
```diff
@@ -9,7 +9,13 @@
...
+    margin-left: 30vh;
+}
+.terms_space1 {
```
Comment (mohit355) [inline frontend/src/components/terms/terms.css:15]: No need for this class. we are using bootstrap classes

File: frontend/src/components/terms/terms.js [hunk 4]
```diff
@@ -13,38 +13,36 @@ const Terms = () => {
...
+          <div >
+              <div>
               </div>
-              <div class="col-lg-8">
-                <h3>Your Agreement</h3>
-                <p>By using this Site, you agree to be bound by, and to comply with, these Terms and Conditions. If you do not agree to these Terms and Conditions, please do not use this site.</p>
-                <p>PLEASE NOTE: We reserve the right, at our sole discretion, to change, modify or otherwise alter these Terms and Conditions at any time. Unless otherwise indicated, amendments will become effective immediately. Please review these Terms and Conditions periodically. Your continued use of the Site following the posting of changes and/or modifications will constitute your acceptance of the revised Terms and Conditions and the resonableness of these standards for notice of changes. For your information, this page was last updated as of the date at the top of these terms and conditions.</p>
-                <p>
+              <div >
+                <h3 className="terms_space">Your Agreement</h3>
```
Comment (mohit355) [inline frontend/src/components/terms/terms.js:20]: no need for new classes just repace the class with the className keyword according to react syntax of writing class

**Review by mohit355 — CHANGES_REQUESTED** [review]
1. First of all, attach the issue that is solved by this PR
2. No need to change the class names. only class keyword change required.
3. Accept and continue button is not it's the right place (because of changing class names)

File: frontend/src/components/terms/terms.js (hunk 4 above)
Comment (kenkirito) [reply frontend/src/components/terms/terms.js:20]: ok

**Comment by Kajol-Kumari** [comment]
closing this PR 'coz of [this](https://github.com/HITK-TECH-Community/Community-Website/pull/186 PR fixing the same issue

**Pull request closed without merging**